helm/values.yaml

```diff
@@ -1,89 +1,111 @@
-# ---------------------------------------------------------------------
-# -- Global values
-# ---------------------------------------------------------------------
+# -- Provide a name in place of `sql-exporter`
+nameOverride: ""
+# -- String to fully override "sql-exporter.fullname"
+fullnameOverride: ""
+
 image:
+  # -- Image repository
   repository: burningalchemist/sql_exporter
+  # -- Image pull policy
   pullPolicy: IfNotPresent
-    #   tag: $IMAGE_TAG
+  # -- Image tag
+  # @default -- `appVersion` value from `Chart.yaml`
+  tag: ""
+
+# -- Secrets with credentials to pull images from a private registry
 imagePullSecrets: []
-nameOverride: ""
-fullnameOverride: ""
+
 service:
+  # -- Service type
   type: ClusterIP
+  # -- Service port
   port: 80
+  # -- Service labels
+  labels: {}
+  # -- Service annotations
   annotations: {}
     # example of prometheus usage
     # prometheus.io/scrape: "true"
     # prometheus.io/path: "/metrics"
-resources:
-  {}
+
+# -- Resource limits and requests for the application controller pods
+resources: {}
   # limits:
   #   cpu: 100m
   #   memory: 128Mi
   # requests:
   #   cpu: 100m
   #   memory: 128Mi
+
+# -- Pod labels
+podLabels: {}
+# -- Pod annotations
+podAnnotations: {}
+
+# -- Pod security context
 podSecurityContext: {}
-securityContext:
-  {}
   # capabilities:
   #   drop:
   #   - ALL
   # readOnlyRootFilesystem: true
   # runAsNonRoot: true
   # runAsUser: 1000
+# @ignored
+securityContext: {}
 
-# ---------------------------------------------------------------------
-# -- Prometheus operator values
-# ---------------------------------------------------------------------
+# Prometheus Operator values
 serviceMonitor:
+  # -- Enable ServiceMonitor
   enabled: true
+  # -- ServiceMonitor interval
   interval: 15s
-  # ---------------------------------------------------------------------
-  # -- SQL exporter configuration
-  # -- For a comprehensive and comprehensively documented configuration
-  # --  file check out: https://github.com/burningalchemist/sql_exporter/blob/master/documentation/sql_exporter.yml
-  # ---------------------------------------------------------------------
+  # -- ServiceMonitor path
+  path: /metrics
+  # -- ServiceMonitor scrape timeout
+  scrapeTimeout: 60s
```

Comment:
This broke things.
From the logs from my prometheus Operator
```
level=warn ts=2023-11-17T04:43:11.936532755Z caller=resource_selector.go:169 component=prometheusoperator msg="skipping servicemonitor" error="scrapeTimeout \"60s\" greater than scrapeInterval \"15s\"" servicemonitor=prometheus/prometheus-sql-exporter namespace=prometheus prometheus=kube-prometheus-stack-prometheus

```
And fires off the `PrometheusOperatorRejectedResources` alert that comes with the Prometheus Operator.
I checked our other ServiceMonitor resources and anywhere the `scrapeTimeout` was set, it was shorter than the `interval`.

Reply:
Hey @jim-barber-he, thanks a lot for reporting! I'll release the fix shortly. 👍